internal/clients/diffreviewer/circleci/circleci_service_test.go

```diff
@@ -353,7 +350,7 @@ func (c *circleCiTestSuite) TestWriteCircleComments() {
 		tp.cs.PrDetails.CommitSha,
 		60,
 	)
-	emptyComments := []*diffreviewer.Comment{}
+	emptyComments := make([]*diffreviewer.Comment, 0)
```

Comment:
Is this a better way to init a slice?

Reply:
it is a weak warning from Goland, they are semantically equivalent, but I guess there may be a slight stylistic preference to do it explicitly with `make` for visibility.

https://stackoverflow.com/a/42917145

i guess maybe the idea is that declaring `var mySlice []string` is semantically different than `mySlice := []string{}`, but at a quick glance they look similar? so adding the `make` helps with clarity?

honestly, i was just clearing IDE warnings, not a big deal 🙂